socketserver/app/srcs/packet/PacketManager.cpp

```diff
@@ -416,7 +411,8 @@ void PacketManager::processPutGame(Poco::Int32 connIndex, char* pBodyData, Poco:
 	sendPacketFunc(connIndex, (char *)&packet, packet.packetSize);
 
 	Poco::Int32 gameIndex = user->getGameIndex();
-	PutInfo putInfo = _gameManager.takeGameByGameIndex(gameIndex)->getPutsBack();
+	Game *game = _gameManager.takeGameByGameIndex(gameIndex);
+	PutInfo putInfo = game->getPutsBack();
 	Poco::Int8 result = _gameManager.checkWinner(gameIndex);
 
 	broadcastPutInfo(roomIndex, putInfo, result);
```

Comment:
여기의 result는 수정하지 않아도 괜찮은 건가요? 확인부탁드립니다:)
```suggestion
	broadcastPutInfo(roomIndex, putInfo, result);
```